consensus/currency.go

```diff
@@ -128,3 +129,9 @@ func (c Currency) MarshalJSON() ([]byte, error) {
 func (c *Currency) UnmarshalJSON(b []byte) error {
 	return c.i.UnmarshalJSON(b)
 }
+
+// Scan implements the fmt.Scanner interface, allowing Currency values to be
+// scanned from text.
+func (c *Currency) Scan(s fmt.ScanState, ch rune) error {
```

Comment:
haha no, it gets read from the ScanState, which is an `io.RuneScanner`. [Relevant source code](http://golang.org/src/math/big/int.go?s=12416:12466#L452)